src/Test/WebTestCase.php

```diff
@@ -568,3 +571,8 @@ protected function tearDown(): void
         parent::tearDown();
     }
 }
+
+// Compatibility layer for Symfony 4.3+
+if (class_exists('Symfony\Bundle\FrameworkBundle\KernelBrowser')) {
+    class_alias('Symfony\Bundle\FrameworkBundle\KernelBrowser', 'Symfony\Bundle\FrameworkBundle\Client');
```

Comment:
hum, well, noooooo
Don't miss reverting depending on how we manage to fix https://github.com/symfony/symfony/issues/31762

Reply:
I will ad it to my todolist to revert in both 2.x and 3.x when the fix comes out

Reply:
Thanks @nicolas-grekas, I created an issue to keep track of this: #520